test/test_type.jl

```diff
@@ -1459,6 +1459,72 @@ isdir(outdir) && rm(outdir, recursive = true)
         end
     end
 
+    @timed_testset "Laplace Diffusion 1D" begin
+        for RealT in (Float32, Float64)
+            equations = @inferred LinearScalarAdvectionEquation1D(RealT(1))
+            diffusivity() = RealT(0.1)
+            equations_parabolic = @inferred LaplaceDiffusion1D(diffusivity(), equations)
```

Comment:
```suggestion
            equations_parabolic = @inferred LaplaceDiffusion1D(RealT(0.1), equations)
```

We used the pattern of defining a function for this only in examples where we needed the parameter several times at different places. That is not the case here, so we can just pass a number to simplify the test (less work for the compiler).